pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -775,39 +775,37 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		switch port.Dhcp {
 		case types.DT_STATIC:
 			if port.AddrSubnet == "" {
-				errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s DT_STATIC but missing "+
-					"subnet address in %+v; ignored", port.IfName, port)
-				log.Error(errStr)
-				port.ParseError = errStr
-				port.ParseErrorTime = time.Now()
+				errStr := fmt.Sprintf("Port %s Configured as DT_STATIC but "+
```

Comment:
DT_STATIC in the code. In the UI we might be calling this just static. Maybe saying "configured as having static IP but missing a subnet address".